common/changes/@itwin/grouping-mapping-widget/uyen-unicode-character_2023-07-14-14-36.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/grouping-mapping-widget",
+      "comment": "Refixed unicode bug to meet OData simple identifier requirement",
```

Comment:
Don't add a separate changelog entry. How can you fix and refix in the same version? In this case you just fixed, not refixed.

Reply:
Oh okay, I see! Sorry, I'm still new to rush. Do I just remove the second changelog?

Reply:
Yes you can remove the file or modify the comment. I do like the more detailed description of this one besides the "Refixed" part.